programs/zstdcli.c

```diff
@@ -672,7 +673,23 @@ int main(int argCount, const char* argv[])
         }
     }
 #endif
-
+    if(operation==zom_list){
+        if(filenameIdx==0){
+            DISPLAY("No files given\n");
+            CLEAN_RETURN(0);
+        }
+        DISPLAY("===========================================\n");
```

Comment:
does `g_displayOut` need to be set each time that `DISPLAY()` is called? I set it equal to `stdout` at the beginning of the block where `zom_list` is detected.